LibreNMS/OS/Powerconnect.php

```diff
@@ -112,4 +116,99 @@ public function pollProcessors(array $processors)
 
         return $data;
     }
+
+    private static function decode_string(string $s): string
+    {
+        $s = str_replace("\n", ' ', $s);
+        $characters = explode(' ', $s);
+
+        $res = '';
+        foreach ($characters as $char) {
+            if (trim($char) === '') {
+                continue;
+            }
+
+            if ($char === '00') {
+                break;
+            }
+            $res .= chr(hexdec($char));
+        }
+
+        return $res;
+    }
+
+    private static function pivotTable(array $table): array
```

Comment:
Hi @vhuk 
Could you open a new PR to remove pivotTable here as well ? 
Thx

Reply:
Done; opened PR #15796 for the change.